src/collectionExporter.ts

```diff
@@ -173,9 +176,12 @@ class CollectionExporter implements IExporter {
 					toUpdate[getPrimary(existing)] = diff;
 				}
 			} else {
-				toInsert.push(lr);
+				if (!opts.skipCreate) toInsert.push(lr);
 			}
 
+			if (opts.forceUpdate && !toUpdate[getPrimary(lr)]) toUpdate[getPrimary(lr)] = lr;
+			if (opts.afterAction) afterActions.push(opts.afterAction);
```

Comment:
Wouldn't you need some status on what had happened (did it update or was it inserted)?
You could pass that along, so the afterAction could do some conditional logic if need be